components/sup/src/manager/service.rs

```diff
@@ -921,12 +902,17 @@ impl Service {
         win_perm::harden_path(path.as_ref())
     }
 
-    fn execute_hooks(&mut self, launcher: &LauncherCli, executor: &TaskExecutor) {
+    /// Returns `true` if the service was restarted or reconfigured.
```

Comment:
This is actually `true` if the service is _marked_ for restarting, or if the reconfigure hook ran (currently synchronously), correct?

Reply:
Yes, that is correct. I will update the comment to be more clear.